app/assets/v2/js/pages/clawback_expired_bounty.js

```diff
@@ -115,6 +115,7 @@ window.onload = function(){
                     e.preventDefault();
                 }
             };
+            // TODO: This call is no longer valid when using Standard Bounties
```

Comment:
yeah.... we will need to implement the standard bounties equiviilent (i think its `.killBounty()`)...

Reply:
Does the Gitcoin UI exist to handle this?

Reply:
yes... gitcoin.co/funding/clawback